src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h

```diff
@@ -148,16 +192,15 @@ namespace OpenMS
 
     // double computeCombinedScore(); // not implemented
     // debug/output functions
-    void outputResults() const;
+    friend OPENMS_DLLAPI std::ostream& operator<<(std::ostream& os, const AccurateMassSearchResult& amsr);
 
 private:
     /// Stored information/results of DB query
     double observed_mz_;
     double theoretical_mz_;
-    double query_mass_;
-    double found_mass_;
+    double observed_mass_;
```

Comment:
I found query mass a little ambiguous since it could be the mass used 
for the query or the mass returned by the query...
Can we find something more clear?


Am 2/4/2015 um 18:47 schrieb Timo Sachsenberg:
>
> In src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h 
> <https://github.com/OpenMS/OpenMS/pull/1232#discussion_r24102805>:
>
> >
> >  private:
> >      /// Stored information/results of DB query
> >      double observed_mz_;
> >      double theoretical_mz_;
> > -    double query_mass_;
> > -    double found_mass_;
> > +    double observed_mass_;
>
> previously, query_mass might have been more correct as it is the mass 
> used to query the database (e.g. neutral mass with adduct removed). 
> observed_mass could be misinterpreted as obtained from observed_mz by 
> simply multiplying by the charge.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OpenMS/OpenMS/pull/1232/files#r24102805>.
>